tests/test_AggregateActionxData.cpp

```diff
@@ -128,7 +128,7 @@ BOOST_AUTO_TEST_CASE (Declared_Actionx_data)
     double secs_elapsed = 3.1536E07;
     const auto ih = Opm::RestartIO::Helpers::
         createInteHead(es, grid, sched, secs_elapsed,
-                       rptStep, rptStep, rptStep);
```

Comment:
This was a totally unrelated change - might induce a data update and should be in a separate PR.

Reply:
This is a code correction that will not trigger a change for the actual test, and has no further implications. I can remove it but cannot promise a separate pull request for it.

Reply:
ok - just leave it